CRM/Contact/BAO/Query.php

```diff
@@ -2341,7 +2341,7 @@ public function restWhere(&$values) {
         $this->_where[$grouping][] = CRM_Core_DAO::createSQLFilter($fieldName, $value, $type);
       }
       else {
-        if (!strpos($op, 'IN')) {
+        if (!self::caseImportant($op)) {
```

Comment:
Hmm so once we do this we have changed from NOT changing IN & NOT IN to changing them but not changing LIKE,  'IS NULL', 'IS NOT NULL', 'IS EMPTY', 'IS NOT EMPTY'

Realistically if anything could have a negative impact it would be 'LIKE' so we are not gaining much here over just removing the whole strtolower line - which has other benefits as well - like improving performance.

We've removed the strotolower in several places already (with tests) without any blow-back. I'd be inclined to remove it and remove the LOWER calls in this function, but also to merge straight after the next rc is cut